app/Http/Controllers/GenreController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\Genre\CreateRequest;
+use App\Http\Requests\Genre\EditRequest;
+use App\Models\Genre;
+use Illuminate\Http\Request;
+
+class GenreController extends Controller
+{
+    public function createForm()
+    {
+        return view('genres.create');
+    }
+
+    public function create(CreateRequest $request)
+    {
+        $data = $request->validated();
+        $genre = new Genre($data);
+        $genre->save();
+
+        session()->flash('success', 'Success!');
+
+        return redirect()->route('genre.list');
+    }
+
+    public function list(Request $request)
+    {
+        $genres = Genre::all();
```

Comment:
тоже пагинация нужна
```suggestion
        $genres = Genre::paginate();
```